Classes/NSManagedObject+ActiveRecord.m

```diff
@@ -366,17 +366,23 @@ - (void)setSafeValue:(id)value forKey:(NSString *)key {
 
     else if ([value isKindOfClass:[NSString class]]) {
 
-        if ([self isIntegerAttributeType:attributeType])
-            value = [NSNumber numberWithInteger:[value integerValue]];
-
-        else if (attributeType == NSBooleanAttributeType)
-            value = [NSNumber numberWithBool:[value boolValue]];
-
-        else if (attributeType == NSFloatAttributeType)
```

Comment:
It seems that the 64 bit changes don't yield any specific caviats. I can add that check as well.